src/components/App/App.js

```diff
@@ -1,62 +1,36 @@
 import React, { lazy, Suspense } from 'react';
 import { IntlProvider } from 'react-intl';
 import { library } from '@fortawesome/fontawesome-svg-core';
-import { BrowserRouter, Route, Switch } from 'react-router-dom';
+import { BrowserRouter } from 'react-router-dom';
 import flatten from 'flat';
 
 import icons from 'assets/icons';
-import AppLocale from 'locales';
 import ErrorBoundary from 'components/ErrorBoundary';
-import Loading from 'components/Loading';
-import ProtectedRoute from 'components/ProtectedRoute';
+import FullPageSpinner from 'components/FullPageSpinner';
+import { useAuthentication } from 'hooks/auth';
 import { useLocale } from 'hooks/locale';
+import AppLocale from 'locales';
 
-import styles from './App.module.scss';
+import './App.module.scss';
 
-const ForgotPasswordPage = lazy(() => import('pages/ForgotPasswordPage'));
-const HomePage = lazy(() => import('pages/HomePage'));
-const NoMatchPage = lazy(() => import('pages/NoMatchPage'));
-const SettingsPage = lazy(() => import('pages/SettingsPage'));
-const SignInPage = lazy(() => import('pages/SignInPage'));
-const SignUpPage = lazy(() => import('pages/SignUpPage'));
+const AuthenticatedApp = lazy(() =>
```

Comment:
what do you think about calling them "GuestApp" and "ProtectedApp"? or some other combination of all of these?

Reply:
I don't find it accurate because we do not have guest pages. The UnauthenticatedApp refers to the Auth flow (sign-in sign-up, forgot password)

Reply:
yes, I was thinking "Guest" as a non-authenticated user. but you are right, if we need to support guest users in the app, that will start giving problems

Reply:
If we want to support guest users rename following your suggestion will work. Probably we also have the guest part of the app always loaded because it is shared between authenticated and unauthenticated users.